extensions/default/src/ViewerLayout/index.tsx

```diff
@@ -37,7 +37,15 @@ function ViewerLayout({
   const navigate = useNavigate();
 
   const onClickReturnButton = () => {
-    navigate('/');
+    let query = new URLSearchParams(window.location.search)
```

Comment:
can you please use const if you are not assigning query? and others? I know it doesn't matter, but the linter would fix it when saved, so better to do it now

Reply:
sure